### ISSUE TITLE
When editing a duplicate article appears at the bottom

### ISSUE THREAD

**Issue by caleb15** [forem/forem#6926]
**Describe the bug**

When editing a article a second article appears after the first.

**To Reproduce**

1. Edit a draft
2. Scroll down
3. There's a duplicate article at the bottom

**Expected behavior**

No duplicate article

**Screenshots**

https://i.imgur.com/H1jkyyr.png

**Desktop (please complete the following information):**

OS: Win10
Browser: Chrome
Version: Version 80.0.3987.149 (Official Build) (64-bit) ~ latest version

**Additional context**

It happens when I edit the article for the first time but after that it disappears.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Hi @caleb15, are you using any extensions? do you see any odd errors in the developer's tools console?

**Comment by caleb15** [comment]
See https://github.com/thepracticaldev/dev.to/issues/6927#issuecomment-605745957 for extensions used. I should've checked the console at the time of the error, sorry. I tried to edit again but didn't get the error.

**Comment by lightalloy** [comment]
While trying to reproduce this and #6927 I ran into another issue #6949 which could be related to these issues (or not).

**Comment by cmgorton** [comment]
From what I can tell this should no longer be an issue with the updated editor. @caleb15 if you run in to this issue again please let me know and I will open this back up.
